lib/ci/container.rb

```diff
@@ -33,11 +35,11 @@ def self.create(connection = Docker.connection,
                     **options_)
       # FIXME: commented to allow tests passing with old containment data
       # assert_version
+      @binds = binds
```

Comment:
this is a class instance variable, previously it was an instance variable. this effectively makes it impossible to have two containers at the same time as they will have the same binds